### ISSUE TITLE
adding components

### ISSUE THREAD

**Issue by cmyr** [linebender/runebender#50]
This is actually fairly tricky, because it requires us to provide UI for selecting the component to add from a list of available glyphs.

A first pass at this would maybe just be a text input field, which would validate on submit.

A simplish nicer option might be a dropdown list of available glyphs; and then there are fancier search-panel options beyond that.

I'm setting this as a goal for the alpha, but I'm willing to push it if it's too much work.

**Comment by madig** [comment]
In Glyphs, you get a popup with a search field and a list displaying the matches and what they look like, below.

**Comment by cmyr** [comment]
yea that sounds like the right UI eventually. We'll want some kind of fuzzy search widget in druid, at any rate :)

**Comment by davelab6** [comment]
I now see Glyph menu, Add Component, but its always deactivated, I see no way to activate it..? :)

**Comment by cmyr** [comment]
Yes, it's not currently connected to anything. This was blocked for a while on better methods for creating windows in druid, it should be possible to get it working now, though.
